<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoReader</class>
 <widget class="QMainWindow" name="VideoReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoReader</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>577</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	color: rgb(31, 28, 28);
	font: 11pt &quot;FontAwesome&quot;;
}

#centralwidget {
	background-color: rgb(31, 28, 28);
}

QWidget {
	background-color: rgb(40, 37, 37);
}

#centralwidget #header {
	height: 20%;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="header" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">screenbtn {
	background-color: rgb(192, 191, 188);
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="title">
         <property name="minimumSize">
          <size>
           <width>561</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: rgb(94, 92, 100);</string>
         </property>
         <property name="text">
          <string>VideoLoader</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="option">
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>38</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(50, 48, 48);
border: none;
border-radius: 5px;
</string>
         </property>
         <property name="text">
          <string extracomment="Click for show options"/>
         </property>
         <property name="icon">
          <iconset theme="folder-new">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QWidget" name="screenbtn" native="true">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
}

QPushButton {
	margin: 0px;
	height: 100%;
	border: none;
	width: 40%;
	color: white;
}
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="reduce">
            <property name="text">
             <string>➖</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="expand">
            <property name="text">
             <string>⬜</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="application-exit">
              <normaloff>.</normaloff>.</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bas" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 28, 28);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>423</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">margin: 0px;
padding: 0px;
background-color: rgb(102, 102, 102);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>121</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWidget" name="progress" native="true">
            <property name="styleSheet">
             <string notr="true">QLabel {
	
	color: rgb(255, 255, 255);
	font: 11pt &quot;FontAwesome&quot;;
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="currTime">
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="videoProgress">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalTime">
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border: none;
}

QFrame {
	position: center;
}

QSlider {
	float: left;
	margin-left: 10px;
}

</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>217</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="maximumSize">
                <size>
                 <width>183</width>
                 <height>61</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border: none;</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pause">
                  <property name="minimumSize">
                   <size>
                    <width>61</width>
                    <height>61</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>61</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset theme="media-playback-pause">
                    <normaloff>.</normaloff>.</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="Play">
                  <property name="minimumSize">
                   <size>
                    <width>61</width>
                    <height>61</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>61</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset theme="media-playback-start">
                    <normaloff>.</normaloff>.</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stop">
                  <property name="minimumSize">
                   <size>
                    <width>61</width>
                    <height>61</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>61</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset theme="media-playback-stop">
                    <normaloff>.</normaloff>.</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>217</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="maximumSize">
                <size>
                 <width>141</width>
                 <height>44</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
font-size: 16pt;</string>
                  </property>
                  <property name="text">
                   <string>🔊</string>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="volumeAjust">
                  <property name="maximumSize">
                   <size>
                    <width>84</width>
                    <height>15</height>
                   </size>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
